rio/commands/detectTag.py

```diff
@@ -0,0 +1,23 @@
+import apriltag
+import cv2 as cv
+import commands2
+
+# is this where this file should go?
+class detectTag(commands2.CommandBase):
+    def __init__(self) -> None:
+        super().__init__()
+        # stuff will go here
+
+    def getCapture(self):
+        cam = cv.VideoCapture(0)
+        capture = cam.read()
+        return capture
```

Comment:
I'd be real worried about having the `rio` do all this work, see the note in the [robotpy docs about camera load](https://robotpy.readthedocs.io/en/stable/vision/roborio.html#on-the-roborio)
And we don't even have a rio2

I would deffo not go this route, if you wanna run all this on the rio, take a look here for ways to do it [NOT in our robot code](https://robotpy.readthedocs.io/en/stable/vision/roborio.html#image-processing)

Alternativly, you can just run everything on a coprocessor like the pi4 we have, plus, keegan is ordering an ai vision accelerator for that pi so you could dual-mode it and run both pipelines together. WPILIB even has a [fork of the official apriltag code](https://github.com/wpilibsuite/apriltag) just for that purpose.